sequel-ace.xcodeproj/project.pbxproj

```diff
@@ -3316,6 +3319,7 @@
 			buildSettings = {
 				APPLY_RULES_IN_COPY_HEADERS = NO;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_GENERATE_ASSET_SYMBOL_FRAMEWORKS = "UIKit AppKit";
```

Comment:
This is the potentially-really-important line

Reply:
We can disable autogeneration completely